### ISSUE TITLE
404 PostSave error after fixing validation error in Nested Content

### ISSUE THREAD

**Issue by rubbad** [umbraco/Umbraco-CMS#9364]
I get a 404 PostSave error after fixing validation errors on mandatory properties in Nested Content. I tried this in a clean Umbraco installation from source 8.9.1

## Umbraco version

I am seeing this issue on Umbraco version: 8.9.1

Reproduction
------------

### Steps to reproduce

* Create an Element Type with one mandatory property
* Create a Document Type with one Nested Content property editor, add the Element type to the allowed content types
* Create the page in the tree, add the element type to the Nested Content list. Fill everything out except the mandatory property of the element type
* Try to save, get the validation error
* Fill in the mandatory property
* Save again

### Expected result

* Expecting the page to be saved

### Actual result
![Screenshot 2020-11-09 at 13 29 26](https://user-images.githubusercontent.com/19747/98541366-9deaa380-228f-11eb-8938-7d9757cd91f9.png)

**Comment by OwainWilliams** [comment]
Hi @raduorleanu - thanks for raising this issue.
 
I've been able to reproduce the bug and as a work around, if you save the new node first, without any nested element set, the validation works after that point. 
It seems to only impact newly created, unsaved nodes.

**Comment by nul800sebastiaan** [comment]
Thank again for the report @rubbad ! I was totally out of it this morning when you asked about this on the other issue, sorry about that! This is indeed exactly the same issue as https://github.com/umbraco/Umbraco-CMS/issues/8954
I was thinking of a similar issue regarding the block list editor, which has been fixed. But I overlooked that it was related to NC! 

I'll close this one, make sure to keep following progress on https://github.com/umbraco/Umbraco-CMS/issues/8954 instead 👍
